build-package.sh

```diff
@@ -10,13 +25,18 @@ export TERMUX_SCRIPTDIR
 # Store pid of current process in a file for docker__run_docker_exec_trap
 source "$TERMUX_SCRIPTDIR/scripts/utils/docker/docker.sh"; docker__create_docker_exec_pid_file
 
+# Functions for working with packages
+source "$TERMUX_SCRIPTDIR/scripts/utils/package/package.sh"
 
 SOURCE_DATE_EPOCH=$(git log -1 --pretty=%ct 2>/dev/null || date "+%s")
 export SOURCE_DATE_EPOCH
 
 : "${TMPDIR:=/tmp}"
 export TMPDIR
 
+# file to update list of compiled packages
+export TERMUX_BUILD_PACKAGE_CALL_BUILT_PACKAGES_LIST_FILE_PATH="${TMPDIR}/build-package-call-built-packages-list-$(date +"%Y-%m-%d-%H.%M.%S.")$((RANDOM%1000))"
```

Comment:
This needs to be set when `TERMUX_BUILD_PACKAGE_CALL_DEPTH=0`, otherwise will be set for each invocation and won't be shared for all depths.